README.md

```diff
@@ -185,3 +185,7 @@ Multiple example apps are included:
 - the `counter` application, which illustrates nonce checking in txs
 - the `dummy` application, which illustrates a simple key-value merkle tree
 - the `dummy --persistent` application, which augments the dummy with persistence and validator set changes
+
+When you are developing an implementation of the ABCI server in your favourite language, we provide the `abci-cli test`
+command that allows you to run our tests against your own ABCI server. Please make sure that your server supports both
+gRPC and the TSP socket protocol.
```

Comment:
Why is it required? One method is sufficient for an ABCI server to work. Also gRPC is slower than raw socket protocol and may not be available in every language. If there’s a TSP implementation, it’s hard to see an use case for running server using gRPC.